### PR TITLE
Add environment variables for CLI options

### PR DESCRIPTION
**Pull request by weilbith**
Based on the suggestions made in #30
Testing seem to make rarely sense, since this is a feature of `Click` we should not verify again by our own.
Closes: #30

### CODE REVIEW

**Comment by weilbith** [comment]
@schmir-at-bb I just added the environment variable remove functionality for the tests. I found such at the `auction-deploy` tool and think it is quite useful/important. I hope this works as expected in practice when it is used within other packages as plugin as well.
We need to update the version by this PR for the `auction-deploy` tools related to PR trustlines-protocol/blockchain#285

File: deploy_tools/cli.py [hunk 1]
```diff
@@ -74,20 +81,22 @@ def validate_address(ctx, param, value):
     help="automatically determine the nonce of first transaction to be sent",
     default=False,
     is_flag=True,
+    envvar="DEPLOY_TOOLS_AUTO_NONCE",
```
Comment (berndbohmeier) [inline deploy_tools/cli.py]: Wouldnt that mean that if I use this for a different command line programm, like `auction-deploy`, that I still have to use `DEPLOY_TOOLS...`. Is that what we want? Or should we provide a way to easy change the prefix?
What do you think @schmir-at-bb?

Comment (schmir-at-bb) [reply deploy_tools/cli.py]: I am fine with the current behaviour. Maybe we can find a better prefix than DEPLOY_TOOLS_ in order to 'decouple' it from the deploy-tools package a bit.

Comment (weilbith) [reply deploy_tools/cli.py]: Yeah, I though about this as well but ended up with not worth the work. But I totally understand the point. Maybe @schmir-at-bb his approach with more decoupled names is already enough. I'm open for suggestions.
I mean the crass approach would be to remove the prefix at all on go for the argument names only. It maybe depends on the use case you had in mind. If you want to have such variables at the top level within your `.zshenv` for (an extreme) example I wouldn't like to generalize them too much. Having them for more convenience on end-to-end test like purpose such simple names work for me.

**Comment by schmir-at-bb** [comment]
> @schmir-at-bb I just added the environment variable remove functionality for the tests. I found such at the `auction-deploy` tool and think it is quite useful/important. I hope this works as expected in practice when it is used within other packages as plugin as well.

To be on the safe side, please consider to replace the call to the function with a session scoped auto-use fixture.
It didn't matter enough back then to me when it was in a conftest.py file, but I think now we should do that.

**Comment by weilbith** [comment]
@schmir-at-bb good point. I have done so as well for the `auction-deploy` tool. :+1:

**Comment by weilbith** [comment]
I need suggestions for better names according to the conversation above. I made the proposal to get rid of the prefix at all (for a focused environment).

**Comment by schmir-at-bb** [comment]
> I need suggestions for better names according to the conversation above. I made the proposal to get rid of the prefix at all (for a focused environment).

I would be fine with dropping the prefix completely.

@berndbohmeier objections?

**Comment by schmir-at-bb** [comment]
@berndbohmeier ping

**Comment by berndbohmeier** [comment]
Yes, we can just drop the prefix.

File: deploy_tools/cli.py (hunk 1 above)
Comment (weilbith) [reply deploy_tools/cli.py]: Decided for deleting the prefix. See comments on this PR below.

**Comment by weilbith** [comment]
@schmir-at-bb feel free to build/upload a new version. Please trigger the #285 then.

**Pull request merged**